endpoints/lookupVoterByIDnumber/app.js

```diff
@@ -4,16 +4,15 @@
 const { Voter, Election, ApiResponse } = require("/opt/Common");
 
 exports.lambdaHandler = async (event, context, callback) => {
+  // Alex:  require firstName and allow blank, or not require at API level?
```

Comment:
Which brings up - if the API request has lastName: "" - will it pass the hasOwnProperty method? Because we want to make sure the "required" fields are not just empty strings.

Reply:
Will refactor and require